pkg/build/gobuild_test.go

```diff
@@ -540,3 +540,70 @@ func TestNestedIndex(t *testing.T) {
 		t.Errorf("Build() expected unexpected mediaType error, got: %s", err)
 	}
 }
+
+func TestGoarm(t *testing.T) {
+	// From golang@sha256:1ba0da74b20aad52b091877b0e0ece503c563f39e37aa6b0e46777c4d820a2ae
+	// and made up invalid cases.
+	for _, tc := range []struct {
+		platform v1.Platform
+		variant  string
+		err      bool
+	}{{
+		platform: v1.Platform{
+			Architecture: "arm",
+			OS:           "linux",
+			Variant:      "vnot-a-number",
+		},
+		err: true,
+	}, {
+		platform: v1.Platform{
+			Architecture: "arm",
+			OS:           "linux",
+			Variant:      "wrong-prefix",
+		},
+		err: true,
+	}, {
+		platform: v1.Platform{
+			Architecture: "arm64",
+			OS:           "linux",
+			Variant:      "v3",
+		},
+		variant: "",
+	}, {
+		platform: v1.Platform{
+			Architecture: "arm",
+			OS:           "linux",
+			Variant:      "v5",
+		},
+		variant: "5",
+	}, {
+		platform: v1.Platform{
+			Architecture: "arm",
+			OS:           "linux",
+			Variant:      "v7",
+		},
+		variant: "7",
+	}, {
+		platform: v1.Platform{
+			Architecture: "arm64",
+			OS:           "linux",
+			Variant:      "v8",
+		},
+		variant: "7",
+	},
+	} {
+		variant, err := getGoarm(tc.platform)
+		if tc.err {
+			if err == nil {
+				t.Errorf("getGoarm expected err for: %v", tc.platform)
```

Comment:
you tripped the Victor linter

Reply:
What do I do??

Reply:
I vehemently disagree with this.

Reply:
cc @vagababov

This has the error message that doesn't show the correct suggestion.

Reply:
`t.Error("getGoarm...:", tc.Platform)` is what it wants

Reply:
I fixed it my own way!